src/Utils/request/uploadFile.ts

```
@@ -43,13 +50,61 @@ const uploadFile = async (
+          options.setOverallProgress(
+            options.totalBytes > 0
+              ? (options.bytesUploadedSoFar / options.totalBytes) * 100
+              : 0,
+          );
+        }
         resolve();
       }
     };
 
-    if (setUploadPercent != null) {
+    if (setUploadPercent != null || (options && options.setOverallProgress)) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)